corehq/apps/reports/standard/forms/reports.py

```diff
@@ -87,6 +103,8 @@ def shared_pagination_GET_params(self):
             name=SubmissionTypeFilter.slug,
             value=[f.type for f in self.submitfilter if f.show]
         ))
+        if self.selected_form_data:
```

Comment:
User can select one form or multiple forms?

Reply:
Then should we use `SingleFormByApplicationFilter` instead?

Reply:
The `SingleFormByApplicationFilter` forces the user to filter by a specific form. My previous comment was maybe a little vague. The user can choose to filter by a single form, but using `FormsByApplicationFilter` allows a user to do a broader filter if needed. Using the `FormsByApplicationFilter` the user is able to partially fill the filter (e.g. only select an app and module, leaving the form section blank) and get a broader set of results.